test/wizard.jl

```diff
@@ -414,6 +414,18 @@ end
     end
 end
 
+@testset "Wizard - state serialization" begin
+    for state_generator in (Wizard.WizardState, step2_state, step3_state, step7_state)
+        mktempdir() do dir
+            state = state_generator()
+
+            Wizard.save_wizard_state(state, dir)
+            @test Wizard.load_wizard_state(dir) == state
```

Comment:
This is untestable as it is now, `Wizard.load_wizard_state` requires interactive input, but this function doesn't take a state as argument.  I take you didn't try it, I don't see how it could possibly work :stuck_out_tongue:

Reply:
[Past Mosè to the rescue](https://github.com/JuliaPackaging/BinaryBuilder.jl/pull/761): we can use the `as_is` keyword argument to skip questions :grin: